tests/scancode/test_ignore_files.py

```diff
@@ -29,13 +29,15 @@
 
 from commoncode.testcase import FileBasedTesting
 from commoncode.ignore import is_ignored
+from scancode.cache import get_scans_cache_class
 from scancode.cli import resource_paths
 from scancode.plugin_ignore import ProcessIgnore
 
 
 class TestIgnoreFiles(FileBasedTesting):
 
     test_data_dir = path.join(path.dirname(__file__), 'data')
+    scan_cache_class = get_scans_cache_class()
```

Comment:
You might want to create a "mock" cache in a test dir to avoid polluting the main cache?